### ISSUE TITLE
realpath:command not found

### ISSUE THREAD

**Issue by BruceGujinhe** [novnc/noVNC#1256]
**Describe the bug**
```
[root@i-cxcizwmr noVNC-master]# ./utils/launch.sh --vnc localhost:5901
Warning: could not find self.pem
Using local websockify at /home/wwwroot/noVNC-master/utils/websockify/run
Starting webserver and WebSockets proxy on port 6080
/home/wwwroot/noVNC-master/utils/websockify/run: line 2: realpath: command not found
/usr/bin/python: websockify is a package and cannot be directly executed
Failed to start WebSockets proxy
```

**To Reproduce**
just run launch.sh in centos 6.8

**Expected behavior**
I found the the content of the file`./utils/websockify/run` is:
```
#!/usr/bin/env sh
BASE_DIR="$(dirname "$(realpath "$0")")"

cd "$BASE_DIR"

python -m websockify $@
```
It will ok after i run `brew install coreutils` with my mac, but it doesn't work in centos.

**Client (please complete the following information):**
 - OS: centos 6.8
 - Browser: chrome
 - Browser version: latest

**Server (please complete the following information):**
 - noVNC version: master
 - VNC server: none, just connect remote virtual machine's host name and port to connect
 - WebSocket proxy: autoload by running `./utils/launch.sh --vnc localhost:5901`

**Comment by BruceGujinhe** [comment]
I found a method to solve it.
just replace `realpath "$0"` with `readlink -e .`,
but I got another issue following:
```
[root@i-cxcizwmr noVNC-master]# ./utils/launch.sh --vnc localhost:5901
Warning: could not find self.pem
Using local websockify at /home/wwwroot/noVNC-master/utils/websockify/run
Starting webserver and WebSockets proxy on port 6080
/usr/bin/python: websockify is a package and cannot be directly executed
Failed to start WebSockets proxy
```

**Comment by BruceGujinhe** [comment]
1. update python to 3.4+
2. `pip install -U websockify`

done

**Comment by CendioOssman** [comment]
Ah. I wasn't aware that realpath was that new. I'll get it fixed. :)

**Comment by CendioOssman** [comment]
Fixed in 6a268a09b668915d6b0e2cd2df1b5425fce1ab92.

**Comment by BruceGujinhe** [comment]
nice work
